README.md

```diff
@@ -119,6 +119,8 @@ Creating a plugin is a rewarding experience! You can choose between first-party
 1. Write your tests
 1. Add your name to the [codeowners](.github/CODEOWNERS) file
 1. Add your plugin to the [Readme](README.md)
+1. Add your plugin to the [autogpt-package](https://github.com/kurtosis-tech/autogpt-package/blob/main/plugins.star). You can copy the line of any of the standard plugins and just add another entry in the dictionary. Raise a PR & get it merged
```

Comment:
Would it be possible to setup an action to check the repo and update this automatically?

Reply:
You mean commit a new line to the autogpt-package when there's a plugin here? We can explore something like that.

At the moment there's a whitelisted set of plugins and if this PR refers to a plugin that isn't in the list of plugins; [we fail the script](https://github.com/kurtosis-tech/autogpt-package/blob/main/main.star#L166)

So if someone is adding a [plugin here](https://github.com/Significant-Gravitas/Auto-GPT-Plugins/blob/master/.github/workflows/test-plugin-installation.yml#L20); they'd get a failure if it isn't in the `plugins.star`.

We could do a couple of things

1. Simplest - Regardless of what we pass in ALLOWLISTED_PLUGINS its the same zip that gets downloaded and installed; maybe just pass one of them? We're duplicating the same steps at the moment with just a different ENV_VAR.
2. I could give you maintainer on AutoGPT-Package (or move it to your name space); so that you can approve both the PR to the package and here at the same time
3. Maybe cleanest- We could have a minimal script here(inside .github maybe or root) that re-uses(imports from the main package) a lot of the AutoGPT-Package but has its own whitelist of plugins that the author can edit while adding a new first party plugin. Kurtosis doesn't need a published package to run, the `main.star` can be called from the CI using `kurtosis run main.star <PARAMS>`